File: projects/code_combat/7_Sarven_Desert/278-Keeping_Time/keeping_time.py
def getItems():
    item = hero.findNearestItem()
    if item:
        hero.moveXY(item.pos.x, item.pos.y)

def manualAttack():
    enemy = hero.findNearestEnemy()
    if enemy:
        hero.attack(enemy)

while True:
    time = hero.time
    if time < 10 or hero.time > 35:
        manualAttack()
    else:
        getItems()

File: projects/code_combat/7_Sarven_Desert/278-Keeping_Time/keepingTime.js
// Use your new skill to choose what to do: hero.time

function getItems() {
    var item = hero.findNearestItem();
    if (item)
        hero.moveXY(item.pos.x, item.pos.y);
}

function manualAttack() {
    var enemy = hero.findNearestEnemy();
    if (enemy)
        hero.attack(enemy)
}

while(true) {
    // If it's the first 10 seconds, attack.
    if (hero.time < 10) {
        manualAttack();
    }
    // Else, if it's the first 35 seconds, collect coins.
    else if (hero.time < 35)
        getItems();
    // After 35 seconds, attack again!
    else
        manualAttack();
}
